app/models/host/managed.rb

```diff
@@ -539,7 +539,7 @@ def jumpstart?
 
   def set_hostgroup_defaults
     return unless hostgroup
-    assign_hostgroup_attributes(%w{environment_id domain_id compute_profile_id})
```

Comment:
Yeah, i was trying to solve http://projects.theforeman.org/issues/9532 in a simple way, but this won't do it :)

Spoke with shimshtein on irc and he is tackling http://projects.theforeman.org/issues/9532 so I'll just revert that anyways